modules/serverless-configuring-kube-rbac-proxy-resources-for-kafka.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * install/kube-rbac-proxy-kafka.adoc
+
+:_content-type: REFERENCE
+[id="serverless-configuring-kube-rbac-proxy-resources-for-kafka_{context}"]
+= Configuring kube-rbac-proxy resources for Kafka
+
+You can globally override resource allocation for the `kube-rbac-proxy` container by using the {ServerlessOperatorName} CR.
+
+[NOTE]
+----
+You can also override resource allocation for a specific deployment.
+----
+
+The following configuration sets Knative Kafka `kube-rbac-proxy` minimum and maximum CPU and memory allocation:
+
+.KnativeKafka CR example
+[source,yaml]
+----
+apiVersion: kafka.knative.dev/v1beta1
```

Comment:
Hmm. IMO, this line should be `apiVersion: operator.serverless.openshift.io/v1alpha1`
Do I miss some recent changes, @skonto ?

Reply:
We use this in CI: https://github.com/openshift-knative/serverless-operator/blob/main/hack/lib/serverless.bash#L349

Reply:
@mgencur I don't know about the `operator.serverless.openshift.io` bit, but `v1alpha1` has been deprecated for a release or two, and in 1.29 it is removed completely.

Reply:
No. It was `operator.knative.dev/v1alpha1` for KnativeServing and KnativeEventing. We also have KnativeKafka which is under different group `operator.serverless.openshift.io`

Reply:
See these files which are up to date for KnativeServing and KnativeEventing: https://github.com/openshift-knative/serverless-operator/tree/main/test/v1beta1/resources

Reply:
Yes Kafka is still in v1alpha. The crds changes are meant for Serving & Eventing.

Reply:
As for the name: https://github.com/openshift-knative/serverless-operator/blob/main/olm-catalog/serverless-operator/manifests/operator_v1alpha1_knativekafka_crd.yaml#L4
it is : `knativekafkas.operator.serverless.openshift.io`.

Reply:
I kind of missed the api version issue, good catch @mgencur

Reply:
(nvm the `v1alpha` deprecation bit -- it doesn't apply here)

Changed to `apiVersion: knativekafkas.operator.serverless.openshift.io/v1alpha1` per Stavros' reply.

Reply:
See https://github.com/openshift/openshift-docs/pull/60660/files#r1214248482 once again please. It has to be `apiVersion: operator.serverless.openshift.io/v1alpha1`